FitnesseRoot/HttpTestSuite/FileServerTestSuite/FourOhFour/content.txt

```diff
@@ -0,0 +1,9 @@
+| script | http browser |
+| set host | localhost |
+| set port | 5000      |
+| get | /no_file_here.txt |
+| ensure | response code equals | 404 |
+| get | /abcdefghijklmnop.jpg |
```

Comment:
I think the existing 404 at `/foobar` has confused more than one apprentice because of there being no file extension. I also figured that a few extra examples would improve clarity and ensure that a 404 response can't be hardcoded for a particular request path.

Reply:
I'm not sure I understand why not having a file extension is confusing. Are apprentices under the impression that files always have extensions?